diff --git a/node_modules/@polkadot/util-crypto/cjs/hd/ledger/index.js b/node_modules/@polkadot/util-crypto/cjs/hd/ledger/index.js
index dce3405..1139ccb 100644
--- a/node_modules/@polkadot/util-crypto/cjs/hd/ledger/index.js
+++ b/node_modules/@polkadot/util-crypto/cjs/hd/ledger/index.js
@@ -11,9 +11,9 @@ function hdLedger(_mnemonic, path) {
         .split(' ')
         .map((s) => s.trim())
         .filter((s) => s);
-    if (![12, 24, 25].includes(words.length)) {
-        throw new Error('Expected a mnemonic with 24 words (or 25 including a password)');
-    }
+    // if (![12, 24, 25].includes(words.length)) {
+    //     throw new Error('Expected a mnemonic with 24 words (or 25 including a password)');
+    // }
     const [mnemonic, password] = words.length === 25
         ? [words.slice(0, 24).join(' '), words[24]]
         : [words.join(' '), ''];
diff --git a/node_modules/@polkadot/util-crypto/hd/ledger/index.js b/node_modules/@polkadot/util-crypto/hd/ledger/index.js
index 78f557b..a22d4c7 100644
--- a/node_modules/@polkadot/util-crypto/hd/ledger/index.js
+++ b/node_modules/@polkadot/util-crypto/hd/ledger/index.js
@@ -8,9 +8,9 @@ export function hdLedger(_mnemonic, path) {
         .split(' ')
         .map((s) => s.trim())
         .filter((s) => s);
-    if (![12, 24, 25].includes(words.length)) {
-        throw new Error('Expected a mnemonic with 24 words (or 25 including a password)');
-    }
+    // if (![12, 24, 25].includes(words.length)) {
+    //     throw new Error('Expected a mnemonic with 24 words (or 25 including a password)');
+    // }
     const [mnemonic, password] = words.length === 25
         ? [words.slice(0, 24).join(' '), words[24]]
         : [words.join(' '), ''];
